source/nodejs/adaptivecards/src/utils.ts

```diff
@@ -130,9 +130,21 @@ export class StringWithSubstitutions {
                     break;
                 }
             }
-
             if (matchedInput) {
-                this._processed = this._processed.replace(matches[0], matchedInput.value ? matchedInput.value : "");
+                var valueForReplace = "";
+                if (matchedInput.value)
+                {
+                    valueForReplace = matchedInput.value;
+                }
+
+                if (contentType == "application/json"){
+                    valueForReplace = JSON.stringify(valueForReplace);
+                    valueForReplace = valueForReplace.slice(1, -1);
+                }else if(contentType == "application/x-www-form-urlencoded"){
```

Comment:
else if on new line, space between if and ( and space between ) and {

Reply:
fixed